src/main/java/it/pagopa/pn/deliverypush/action/startworkflow/notificationvalidation/NotificationValidationActionHandler.java

```diff
@@ -60,10 +66,19 @@ public void validateNotification(String iun, NotificationValidationActionDetails
                 taxIdPivaValidator.validateTaxIdPiva(notification);
             }
 
-            //La validazione dell'indirizzo è async
-            MDCUtils.addMDCToContextAndExecute(
-                    addressValidator.requestValidateAndNormalizeAddresses(notification)
-            ).block();
+            if (f24Exists(notification)) {
+                //La validazione del F24 è async
+                MDCUtils.addMDCToContextAndExecute(
+                        f24Validator.requestValidateF24(notification)
+                ).block();
+            } else {
+                String detail = " F24 does not exists, so F24 validation will be skipped";
+                generateSkipAuditLog(notification, SECOND_VALIDATION_STEP, detail);
```

Comment:
inserito qui l'auditlog, mi sa che vien stampato tra il "before" 1 e il generateSuccess 1
(mentre quando esegue la request validate, il "2", vien stampato in risposta dalla coda, quindi comparirà dopo)
se non è un problema, fare un pò di refactor per farli comparire nell'ordine corretto